sdks/java/io/kafka/pom.xml

```diff
@@ -106,7 +106,7 @@
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka-clients</artifactId>
-      <version>0.9.0.1</version>
+      <version>[0.9.0.1,0.11.0)</version>
```

Comment:
sure, would roll back to version '0.9.0.1' here. noticed that it's version 0.10.* with this range.